lib/json/json.go

```diff
@@ -23,6 +24,16 @@ import (
 	"go.starlark.net/starlarkstruct"
 )
 
+func getPointer(i interface{}) (uintptr, bool) {
+	v := reflect.ValueOf(i)
+	switch v.Kind() {
+	case reflect.Ptr, reflect.Chan, reflect.Map, reflect.UnsafePointer, reflect.Slice:
+		return v.Pointer(), true
```

Comment:
Let's use v.UnsafePointer() and type unsafe.Pointer instead, since we don't need the numeric value of the pointer and we're not trying to hide anything from the GC.

Reply:
Done